catchup/peerSelector.go

```diff
@@ -391,7 +410,7 @@ func (ps *peerSelector) peerDownloadDurationToRank(psp *peerSelectorPeer, blockD
 // addToPool adds a given peer to the correct group. If no group exists for that peer's rank,
 // a new group is created.
 // The method return true if a new group was created ( suggesting that the pools list would need to be re-ordered ), or false otherwise.
-func (ps *peerSelector) addToPool(peer network.Peer, rank int, class peerClass, peerHistory *historicStats) bool {
+func (ps *PeerSelector) addToPool(peer network.Peer, rank int, class peerClass, peerHistory *historicStats) bool {
```

Comment:
You don't need to export this function. Do you?

Reply:
nope